dashboard/test/controllers/api/v1/sections_students_controller_test.rb

```diff
@@ -81,6 +81,44 @@ class Api::V1::SectionsStudentsControllerTest < ActionController::TestCase
     assert @student.valid_password?('testpassword')
   end
 
+  test 'teacher can reset secret picture and words for their student' do
+    sign_in @teacher
+    @student.reload
```

Comment:
This strikes me as a _little_ odd, but I understand why it's here (because the student is shared between tests).  I'd personally be in favor of changing that `setup_all` into a `setup` block, but understand if you don't want to tackle that in this PR.

Reply:
We could also leave the creation in `setup_all` and add this reload to the `setup`. Then again maybe that's also confusing and it's best just to recreate the student before each test case. If you really want to get dirty, you could benchmark each approach and see which one is faster ;)

Reply:
i'll add a follow-up task for myself for this along with the put-to-patch change for these tests!